administrator/language/en-GB/guidedtours.joomla_welcome_steps.ini

```
@@ -0,0 +1,14 @@
+COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_MENUS_TITLE="The menu"
```

```suggestion
COM_GUIDEDTOURS_TOUR_WELCOMETOJOOMLA_STEP_MENUS_TITLE="The Menu"
```

Follows the style guide